helm/templates/arbiter/statefulset-arbiter-rs.yaml

```diff
@@ -61,17 +75,15 @@ spec:
                   fieldPath: metadata.name
             {{- if .Values.usePassword }}
```

Comment:
```suggestion
            {{- if and .Values.replicaSet.enabled (eq .Values.config.security.authorization "enabled") }}
```

Reply:
not adding the `.Values.replicaSet.enabled` check as it is already present as a global check in the file